### PR TITLE
bc: add calculator page

### PR DESCRIPTION
**Pull request by igorshubovych**

### CODE REVIEW

**Comment by felixonmars** [comment]
:+1:

File: pages/common/bc.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# bc
+
+> Calculator.
+
+- Run calculator in the interactive mode
+
+`bc -i`
+
+- Calculate the result of expression
```
Comment (waldyrious) [inline pages/common/bc.md]: Perhaps "of an expression"?

File: pages/common/bc.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# bc
+
+> Calculator.
+
+- Run calculator in the interactive mode
```
Comment (waldyrious) [inline pages/common/bc.md]: I think the "the" is unnecessary here.

Comment (ostera) [inline pages/common/bc.md]: I agree

File: pages/common/bc.md (hunk 1 above)
Comment (ostera) [inline pages/common/bc.md]: I agree

**Comment by igorshubovych** [comment]
I fixed grammar mistakes.

@waldyrious thanks for feedback!

My English is bad. This is exactly why I always submit pull requests to hear your feedback.

**Pull request merged**